payments/management/commands/update_expired_orders.py

```diff
@@ -0,0 +1,18 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db.transaction import atomic
+
+from payments.models import Order
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Sets too old orders from state "waiting" to state "expired."'
```

Comment:
I suppose the dot at the end near "expired" should be outside of the quotation marks.

Reply:
👍